packages/core/components/AutoField/fields/SelectField/index.tsx

```diff
@@ -2,6 +2,7 @@ import getClassNameFactory from "../../../../lib/get-class-name-factory";
 import styles from "../../styles.module.css";
 import { ChevronDown } from "lucide-react";
 import { FieldPropsInternal } from "../..";
+import {safeJsonParse } from "../../../../lib/safe-json-parse"
```

Comment:
Can you make sure you're running prettier or run `npm run format` and commit the results? Else CI will fail.

Reply:
Ah yeah! weirdly vs-code didn't pick up the prettier config, later (one day) will look at adding a pre-commit hook to auto do this, after the real work is done, but did this manually no problemo was bothering me too

Reply:
Please no pre commit hooks 😅 really disrupts my workflow

Reply:
ok done... or not done... I mean  hahah no problem.